docs/publications.rst

```diff
@@ -4,25 +4,65 @@ Papers, Presentations, and Posters
 Publications about or related to using LBANN:
 
 + Nikoli Dryden, Naoya Maruyama, Tom Benson, Tim Moon, Marc Snir,
-  Brian Van Essen. "Channel and Filter Parallelism for Large-Scale
-  CNN Training", to appear in *International Conference for High
-  Performance Computing, Networking, Storage and Analysis (SC'19)*, 2019.
-
-+ Sam Ade Jacobs, Brian Van Essen, Tim Moon, Jae Seung Yeom, David
-  Hysom, Brian Spears, Rushil Anirudh, Jayaraman Thiagaranjan, Shusen
-  Liu, Jim Gaffney, Peer-Timo Bremer, Tom Benson, Peter Robinson, and
-  Luc Peterson, "Parallelizing Training of Deep Generative Models on
-  Massive Scientific Datasets", to appear in *Proceedings of Cluster
-  Computing*, 2019
+  Brian Van Essen. `"Channel and Filter Parallelism for Large-Scale
+  CNN Training" <https://dl.acm.org/doi/10.1145/3295500.3356207>`_, in
+  *SC '19: Proceedings of the International Conference for High
+  Performance Computing, Networking, Storage and Analysis*, November 2019
+  Article No. 10, Pages 1-20,
+  `https://doi.org/10.1145/3295500.3356207 <https://dl.acm.org/doi/10.1145/3295500.3356207>`_
+  :ref:`BibTex <bib/dryden2019.rst>`
+
++ Sam Ade Jacobs, Brian Van Essen, David Hysom, Jae Seung Yeom, Tim
+  Moon, Rushil Anirudh, Jayaraman Thiagaranjan, Shusen Liu, Jim
+  Gaffney, Peer-Timo Bremer, Tom Benson, Peter Robinson, and Luc
+  Peterson, Brian Spears, `"Parallelizing Training of Deep Generative
+  Models on Massive Scientific
+  Datasets" <https://doi.org/10.1109/CLUSTER.2019.8891012>`_, in `*2019
+  IEEE International Conference on Cluster Computing
+  (CLUSTER)* <https://clustercomp.org/2019/program/>`_, Albuquerque,
+  NM, USA, 2019, pp. 1-10, doi:
+  `10.1109/CLUSTER.2019.8891012<https://doi.org/10.1109/CLUSTER.2019.8891012>_`.::
+
+    @INPROCEEDINGS{8891012,
```

Comment:
I assume you're going to move this into it's own `rst` file? Also, the keys should be consistent.

Reply:
Yes, I am trying to get one to work.